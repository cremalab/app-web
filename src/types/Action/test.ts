import { Action } from "."

describe("Action", () => {
  it("returns an `Action<Type, Payload>`", () => {
    // Arrange
    const type = "SOME_TYPE"
    const payload = 2

    // Act
    const received = Action(type, payload)

    // Assert
    expect(received).toMatchSnapshot()
  })
})
