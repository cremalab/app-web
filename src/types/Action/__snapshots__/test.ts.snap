// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Action returns an \`Action<Type, Payload>\` 1`] = `
Object {
  "payload": 2,
  "type": "SOME_TYPE",
}
`;
